src/ServicePulse.Host/vue/src/components/failedmessages/AllDeletedMessages.vue

```diff
@@ -115,10 +115,10 @@ function loadPagedMessages(groupId, page, sortBy, direction, startDate, endDate)
   return loadDelMessagesPromise;
 }
 
-function updateMessagesScheduledDeletionDate(messages) {
+function updateMessagesScheduledDeletionDate(messages: FailedMessageViewWithExtendedUIProperties[]) {
   //check deletion time
-  messages.forEach((message) => {
-    message.error_retention_period = moment.duration(configuration.value.data_retention.error_retention_period).asHours();
+  messages.forEach((message: FailedMessageViewWithExtendedUIProperties) => {
```

Comment:
explicit type shouldn't be required here, since messages is a known type

Reply:
yeah, good catch, that is something I left behind when I was attempting using an union type for addressing the same thing you mentioned in this other comment https://github.com/Particular/ServicePulse/pull/1769#discussion_r1515400217